lib/katello/plugin.rb

```diff
@@ -164,4 +164,11 @@
 
   register_custom_status(Katello::ErrataStatus)
   register_custom_status(Katello::SubscriptionStatus)
+
+  extend_page "smart_proxies/show" do |cx|
+    cx.add_pagelet :details_content,
+                   :name => _('Storage'),
+                   :partial => 'smart_proxies/show/storage',
+                   :onlyif => Proc.new { |proxy| proxy.has_feature?(SmartProxy::PULP_FEATURE) }
```

Comment:
You probably want to check for the PULP_NODE_FEATURE feature as well.  We use that to differentiate between the 'built in' capsule and remote capsules.